cobalt/media/base/sbplayer_interface.h

```diff
@@ -37,12 +38,21 @@ class SbPlayerInterface {
       SbPlayerStatusFunc player_status_func,
       SbPlayerErrorFunc player_error_func, void* context,
       SbDecodeTargetGraphicsContextProvider* context_provider) = 0;
+  virtual SbPlayer Create(
+      SbWindow window,
+      const StarboardExtensionMaxInputSizePlayerCreationParam* creation_param,
+      SbPlayerDeallocateSampleFunc sample_deallocate_func,
+      SbPlayerDecoderStatusFunc decoder_status_func,
+      SbPlayerStatusFunc player_status_func,
+      SbPlayerErrorFunc player_error_func, void* context,
+      SbDecodeTargetGraphicsContextProvider* context_provider) = 0;
```

Comment:
We can always support setting max input size and have only one Create() function in SbPlayerInterface.h, and check if there's an extension available in SbPlayerInterface.cc. So, we remove a few redundant code.

Reply:
Changed it with one Create().